[debug] [2024-01-26T17:55:49.503Z] ----------------------------------------------------------------------
[debug] [2024-01-26T17:55:49.507Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Admin\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js deploy
[debug] [2024-01-26T17:55:49.508Z] CLI Version:   12.9.1
[debug] [2024-01-26T17:55:49.508Z] Platform:      win32
[debug] [2024-01-26T17:55:49.509Z] Node Version:  v18.16.0
[debug] [2024-01-26T17:55:49.510Z] Time:          Fri Jan 26 2024 23:55:49 GMT+0600 (Bangladesh Standard Time)
[debug] [2024-01-26T17:55:49.510Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-01-26T17:55:49.516Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2024-01-26T17:55:49.733Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2024-01-26T17:55:49.734Z] > authorizing via signed-in user (mdnomanm622@gmail.com)
[debug] [2024-01-26T17:55:49.734Z] [iam] checking project my-portfolio-site-52fab for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2024-01-26T17:55:49.736Z] > refreshing access token with scopes: []
[debug] [2024-01-26T17:55:49.738Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-01-26T17:55:49.738Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-01-26T17:55:49.759Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: getaddrinfo ENOTFOUND firebase-public.firebaseio.com
[warn] !  Unable to fetch the CLI MOTD and remote config. 
[debug] [2024-01-26T17:55:49.763Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2024-01-26T17:55:49.764Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------340542786268601876819411","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-01-26T17:55:49.764Z] [iam] error while checking permissions, command may fail: FirebaseError: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2024-01-26T17:55:49.765Z] > refreshing access token with scopes: []
[debug] [2024-01-26T17:55:49.765Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-01-26T17:55:49.766Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-01-26T17:55:49.782Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------120676699422063005681217","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-01-26T17:55:49.783Z] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[error] 
[error] Error: Assertion failed: resolving hosting target of a site with no site name or target name. This should have caused an error earlier
